### DISCUSSION TITLE
和win11的实时字幕相比哪个好?

### DISCUSSION THREAD

**Discussion by HexWeaverNova** [jxlpzqc/TMSpeech#8] [Q&A]
和win11的实时字幕的准确率哪个高, 哪个反应更快?

**Comment by am009** [comment]
我还没试过win11的实时字幕，有没有人放一起对比试试  😆

**Comment by kiFte** [comment]
自带模型咱自己测下来是win11好，但win11自带的那玩意会不时出现字幕缺失...

**Comment by WX66JY** [comment]
旧版本和新的不能用这是为啥啊
![Uploading f8ba201e979fa250004b179d9a6c11cc.png…]()
